WcaOnRails/app/webpacker/lib/wca-data.js.erb

```diff
@@ -138,3 +138,7 @@ export const councilsStatus = {
   senior_member: 'senior_member',
   leader: 'leader',
 };
+
+// ----- PANEL LIST -----
+
+export const PANEL_LIST = JSON.parse('<%= PanelController.panel_list.to_json.html_safe %>');
```

Comment:
You don't need to parse JSON through `JSON.parse`. It is called "JavaScript Object Notation" because, well, it's designed to match the way JavaScript parses objects.

If you drop the string quotes and inject the body directly like so: `export const PANEL_LIST = <%= RailsStuff.to_json.html_safe %>` (note the absence of string tick marks around the `<%= %>`) it should work

Reply:
Oh yes, that worked. Thanks.